mlir/test/Dialect/Linalg/transform-op-fuse-into-containing.mlir

```diff
@@ -26,8 +26,8 @@ module {
       // CHECK: %[[T1:.*]] = linalg.fill {{.*}} outs(%[[T0]]
       %6 = tensor.extract_slice %0[%3] [%4] [1] : tensor<?xf32> to tensor<?xf32>
 
-      // CHECK: %[[T2:.*]] = linalg.elemwise_unary ins(%[[T1]]
-      %7 = linalg.elemwise_unary ins(%6 : tensor<?xf32>) outs(%5 : tensor<?xf32>) -> tensor<?xf32>
```

Comment:
What is/was the default for `linalg.elemwise_unary` when `fun` is not provided? Is it the `0`th enum item, i.e. `UnaryFn::exp`?

Reply:
Hopefully the new `linalg.elementwise` does not have this "feature".

Reply:
yes, `exp` for unary and `add` for binary. I believe we explicitly removed that "feature" from the new element wise